support/proxy/vwf.example.com/jplayer/videoManager.vwf.yaml

```diff
@@ -6,8 +6,14 @@ properties:
   loop: false
   containerDivId: "videoContainer"
   playerDivId: "videoScreen"
+  containerSize: [ 1024, 768 ]
+  playerSize: [1024, 768]
```

Comment:
As a general web design rule of thumb, I don't like that the application is setting hard restrictions on the size of the player. The browser window can be resized and of course we have varying screen resolutions to consider. What is the purpose of setting the size like this?